pkg/services/sqlstore/migrations/migrations_test.go

```diff
@@ -281,14 +281,23 @@ func getDBType() string {
 	return dbType
 }
 
-func getTestDB(dbType string) sqlutil.TestDB {
+func getTestDB(t *testing.T, dbType string) sqlutil.TestDB {
 	switch dbType {
 	case "mysql":
 		return sqlutil.MySQLTestDB()
 	case "postgres":
 		return sqlutil.PostgresTestDB()
```

Comment:
These are only in play when you use the `GRAFANA_TEST_DB=... go test ...` right? Nothing in CI using this?

Reply:
correct! also mentioned [here](https://github.com/grafana/grafana/pull/44101#discussion_r785965814)